python/Framework.py

```diff
@@ -328,15 +307,12 @@ def redoJEC(self, JECDatabase=None, addBtagDiscriminators=None):
 
         print("")
 
-        self.__jec_done = True
-
+    @dep(before=("create", "jer"), performs=("correction", "jer"))
     def smearJets(self, resolutionFile=None, scaleFactorFile=None):
```

Comment:
would it be possible to keep in sync what the function does (smear jets) and the name of the action performed (jer) ? This may ease debugging :smile:

Reply:
Sure, I used the `__jer_done` to get a name, but this is easy to change. On the other hand, the "action" needs not to say exactly what the function does (it's only used for ordering things logically). Not an expert here, "smearJets" also sounds good to me

Reply:
ok, let's keep jer :)